packages/ember-toucan-core/src/components/form/fields/input.ts

```diff
@@ -59,7 +83,29 @@ export default class ToucanFormInputFieldComponent extends Component<ToucanFormI
   }: AssertBlockOrArg) =>
     assertBlockOrArgumentExists({ blockExists, argName, arg, isRequired });
 
+  @action
+  handleCount(event: Event | InputEvent): void {
+    assert(
+      'Expected HTMLInputElement',
+      event.target instanceof HTMLInputElement
+    );
+
+    if (event.target?.value) {
+      this.count = event.target.value.length;
+    } else {
+      this.count = 0;
+    }
```

Comment:
At the end a matter of taste, but another way to write this is:

```suggestion
    this.count = event.target?.value.length ?? 0;
```